plugins/scaffolder-backend/src/service/router.test.ts

```diff
@@ -895,6 +895,11 @@ data: {"id":1,"taskId":"a-random-id","type":"completion","createdAt":"","body":{
       it('filters steps that the user is not authorized to see', async () => {
         jest
           .spyOn(permissionApi, 'authorizeConditional')
+          .mockImplementationOnce(async () => [
+            {
+              result: AuthorizeResult.ALLOW,
+            },
+          ])
```

Comment:
Actually this is needed because the `checkPermissions` utility function I added will throw a 403 error if there contains a `AuthorizeResult.DENY` in the list of permission responses, so I needed to add this.

Reply:
No longer needed after converting the permission to a `BasicPermission` and using `authorize` instead of `authorizeConditional` for the `checkPermissions` utility function.